# A message type to contain 1D numpy arrays and the current position of the UAV

# time stamp
int64 rows
int64 cols
float64 pos_x
float64 pos_y
float64[] data
int64 r_max
int64 ang_bins
